internal/dashboard/handler/alarm_handler.go

```diff
@@ -257,7 +292,7 @@ func DeleteReceiver(_ *gin.Context) (any, error) {
 // @Router /api/v1/alarm/receiver/templates [POST]
 // @Security ApiKeyAuth
 func ListReceiverTemplates(_ *gin.Context) ([]receiver.Template, error) {
-	return nil, httpErr.NewNotImplemented("not implemented")
+	return alarm.ListReceiverTemplates()
```

Comment:
The `context.Context` parameter should be passed in.

Reply:
there's no need to pass the context